pkg-manager/plugin-commands-installation/test/update/interactive.ts

```diff
@@ -65,6 +65,11 @@ test('interactively update', async () => {
     value: '',
   }
 
+  await Promise.all([
+    addDistTag({ package: 'is-negative', version: '2.1.0', distTag: 'latest' }),
+    addDistTag({ package: 'micromatch', version: '4.0.0', distTag: 'latest' }),
+  ])
```

Comment:
Cannot pin `is-positive` because `addDistTag` would attempt to remove `latest` from `is-positive` which doesn't exist.